src/bootstrap/builder.rs

```diff
@@ -574,6 +574,9 @@ impl<'a> Builder<'a> {
             cargo.env("RUSTC_SAVE_ANALYSIS", "api".to_string());
         }
 
+        // For `cargo doc` invocations, make rustdoc print the Rust version into the docs
+        cargo.env("RUSTDOC_CRATE_VERSION", self.build.rust_version());
```

Comment:
Let's also add this to `rustdoc_cmd` above to make certain it's passed in all cases.

Reply:
Force-pushed with this update.